DigitalPuzzle/DigitalPuzzle.ino

```diff
@@ -0,0 +1,343 @@
+﻿/*******************************************
+* function:get the id of RFID key
+* RFID   Uno r3ú
+* VCC    3.3V
+* RST    2
+* GND    GND
+* MISO   3
+* MOSI   4
+* SCK    5
+* NSS    6
+* IRQ    7
+****************************************/
+
+#include <EEPROM.h>
+#include"rfid1.h"
+
+const int RedPin[2] = {3,4};
+const int GreenPin[2] = {2,5};
+
+int count = 0;
+
+
+RFID1 rfid;		 //Create a variable type to read the RFID scanner 
+
+bool StartValues[6] = {1,1,1,0,0,0};
+bool ReaderResult[8] = {0,0,0,0,0,0,0,0};
+
+uchar serNum[5]; //Array to store your 
```

Comment:
Gebruik goede variabel namen. Ik denk dat hier serialNumber bedoelt wordt? Het commentaar is ook niet heel lekker hier :)